CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+cmake_minimum_required(VERSION 3.15)
+
+project(yc-libvhost-server)
+
+if(UNIX AND NOT APPLE)
+  set(LINUX TRUE)
+endif()
+
+if(NOT LINUX)
+  message(FATAL_ERROR "Unsupported platform")
+endif()
+
+SET(libvhost_log_verbosity "LOG_INFO" CACHE STRING "Libvhost log verbosity")
```

Comment:
Can we settle on one case here, everything else is lower so probably just `set` is fine

Whereas the variable itself should probably be uppercase, aka `LIBVHOST_LOG_VERBOSITY`

Reply:
Fixed